### PR TITLE
[#7562] Deprecate msiSendMail (main)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

**Comment by trel** [comment]
looks good

File: server/re/src/mailMS.cpp [hunk 1]
```diff
@@ -14,12 +14,16 @@
  *
  * \brief Sends email
  *
+ * This microservice is deprecated and will be removed in a later release. Consider using the Python rule engine plugin
+ * or exposing your own mail utility via msiExecCmd.
```
Comment (alanking) [inline server/re/src/mailMS.cpp:18]: Consider using the `\deprecated` tag?

**Review by alanking — COMMENTED** [review]
We used to log a message for the microservices, but... I don't see much value in that, so I think this is good. Just had one suggestion

File: server/re/src/mailMS.cpp (hunk 1 above)
Comment (korydraughn) [reply server/re/src/mailMS.cpp:18]: Will do.

**Review by alanking — COMMENTED** [review]
Seems good to me. Squash em

**Comment by korydraughn** [comment]
Squashed and pounded.

**Review by alanking — APPROVED** [review]

**Pull request merged**